tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p>This section is not necessary. There is nothing special about training with clustering, only defining the model. </p><p> </p><p>See how QAT describes it. There is no section and the overview says "Training is otherwise standard."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>